Libraries/Components/TextInput/TextInput.js

```diff
@@ -673,6 +690,12 @@ const TextInput = React.createClass({
     }
   },
 
+  _onContentSizeChange(event: Event) {
+    if (this.props.onContentSizeChange) {
+      this.props.onContentSizeChange(event.nativeEvent.contentSize);
```

Comment:
Yea I wasn't sure about this, I did like `onChangeText` where we pass only the text. Personally I prefer having only the relevant value passed the the callback instead of the whole event in case where the event is pretty specific.

With that said I don't mind changing this to passing the event either as I can see it has some advantages and we may use this pattern more often.